### DISCUSSION TITLE
No improvement of finetune my own data

### DISCUSSION THREAD

**Discussion by JasonSloan** [xinntao/Real-ESRGAN#701] [Q&A]
I use the 'finetune_realesrgan_x4plus_pairdata.yml' configuration file, and I only train the real-ESRGANx4 model instead of training the real-ESRNetx4 model first. However, I cant get a better result of finetune the model on validation dataset. The total number of my training dataset is about 6000 images while the number of validation dataset is about 800. The average psnr value is always about 24 no matter how many epochs I trained. Should I train the real-ESRNetx4 first, or should I use the multi scale script to make the data more abundant? Or should I lower the learning rate? Here's my configuration file looks like:

[finetune_realesrgan_x4plus_pairdata.txt](https://github.com/xinntao/Real-ESRGAN/files/12842559/finetune_realesrgan_x4plus_pairdata.txt)
